pkg/sql/execinfrapb/processors_bulk_io.proto

```diff
@@ -134,6 +134,10 @@ message ReadImportDataSpec {
   // NEXTID: 16
 }
 
+message StreamIngestionDataSpec {
+  map<int32,string> partition_address = 1;
```

Comment:
I think we can also add `[(gogoproto.castvalue) = "github.com/cockroachdb/cockroach/pkg/ccl/streamingccl/streamclient.PartitionAddress"];` on this line to get this map to be typed a bit stricter. I also think we can `castkey` to a `PartitionID` type that I can add to the `streamclient` package.

Reply:
changed it to a repeated field as that is what the ingestion processor PR changed it to.